metadata/water/lakes_nhd.md

```diff
@@ -0,0 +1,83 @@
+# Title
+
+Utah Lakes NHD
+
+## ID
+
+b0a55056-bdb9-4b7c-8812-8f246b328f88
+
+## Brief Summary
+
+Statewide areal bodies of water such as lakes, reservoirs, and ponds derived from the National Hydrography Dataset (NHD).
+
+## Summary
+
+This dataset comprises bodies of water found in and around the state of Utah. This dataset is derived from the [National Hydrography Dataset](https://www.usgs.gov/national-hydrography/national-hydrography-dataset) (NHD) compiled by the United State Geological Survey [USGS](https://www.usgs.gov/) and modified by UGRC to provide additional functionality for Utah-based users.
+
+## Description
+
+### What is the dataset?
+
+The NHD is a national framework for assigning [reach addresses](https://enviro.epa.gov/enviro/ef_metadata_html.tri_page?p_column_name=reach_code#:~:text=Description%3A%20A%20reach%20code%20is,National%20Hydrography%20Dataset%20(NHD).) to water-related entities, such as industrial discharges, drinking water supplies, and fish habitat areas. Reach addresses establish the locations of these entities relative to one another within the NHD surface water drainage network, much like addresses on streets.
```

Comment:
This feels like a lot of text to establish reach codes to then finally establish what the polygons are. Is there a way we could invert this, so we lead off with what the polygons are and then talk about reach codes at the end if it's necessary?

Reply:
I've moved some things around, let me know what you think! I'll probably keep fiddling with it as I work on Streams as well (since their metadata are so similar).

Reply:
It definitely reads better now. Thank you!